dkist/io/asdf/converters/tiled_dataset.py

```diff
@@ -5,16 +5,22 @@ class TiledDatasetConverter(Converter):
     tags = [
         "tag:dkist.nso.edu:dkist/tiled_dataset-0.1.0",
         "asdf://dkist.nso.edu/tags/tiled_dataset-1.0.0",
+        "asdf://dkist.nso.edu/tags/tiled_dataset-1.1.0",
```

Comment:
In general with this schema bump I think we should standardise across Dataset and TiledDataset both in memory and on disk.